upstairs/src/lib.rs

```diff
@@ -4231,6 +4268,7 @@ impl Downstairs {
                             self.downstairs_errors
                                 .insert(client_id, errors + 1);
                         }
+
```

Comment:
Nit: looks like an extra blank line slipped in here..

Reply:
That was an intention separation, but I added a comment in 940cacb